davdwhyte87/iReporter
# #162489615 Api implement admin change status

**What does this PR do?**
This pull request implements admin change status functionality

**Description of task to be completed?**
- check if user is an admin for updates
- update status if the user is an admin

**How ca this be manually tested?**
- clone the repo
- cd into the project directory
- run npm i to install dependencies
- run npm run create-db to setup database (postgresql should be installed with a databse `ireporter`
- run npm start
- send PATCH request to endpoint `/api/v1/red-flag/:id`  or `/api/v1/intervention/:id` with the new
 status and a token from an admin.

**Relevant pivotal tracker stories**
`#162489615`
